uaa/src/main/webapp/resources/predix/stylesheets/predix-styles.css

```diff
@@ -208,15 +210,13 @@ h1 {
 
 .header {
   margin: 0 auto 1rem;
+  width: 100%;
   height: 60px;
   background: no-repeat center;
   background-size: contain; }
-  @media (max-width: 51.13333rem) {
-    .header {
-      background-size: 150px; } }
-  @media only screen and (min-width: 321px) and (max-width: 1024px) {
-    .header {
-      background-size: 300px; } }
+
+@media only screen and (min-width: 321px) and (max-width: 1024px) {
```

Comment:
this block doesn't seem to do anything, remove the whole thing

Reply:
yea I GUESS we can remove it. X)